I/O device error on read file on windows.

**What happened**:
https://github.com/juicedata/juicefs/actions/runs/10320840411/job/28572330782

Directory: Z:\

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----            8/9/2024  2:25 PM                dir

    Directory: Z:\dir

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-----            8/9/2024  2:25 PM              7 hello.txt
Get-Content: D:\a\_temp\9073478a-df[14](https://github.com/juicedata/juicefs/actions/runs/10320840411/job/28572330782#step:16:15)-4886-8817-040ffd828e00.ps1:6
Line |
   6 |  type "Z:\dir\hello.txt"
     |  ~~~~~~~~~~~~~~~~~~~~~~~
     | The request could not be performed because of an I/O device error. : 'Z:\dir\hello.txt'
Error: Process completed with exit code 1.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version:
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Object storage (cloud provider and region, or self maintained):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

--- a/pkg/vfs/vfs.go
+++ b/pkg/vfs/vfs.go
@@ -743,7 +743,7 @@ func (v *VFS) Read(ctx Context, ino Ino, buf []byte, off uint64, fh uint64) (n i
 	}
 
 	// there could be read operation for write-only if kernel writeback is enabled
-	if !v.Conf.FuseOpts.EnableWriteback && !hasReadPerm(h.flags) {
+	if v.Conf.FuseOpts != nil && !v.Conf.FuseOpts.EnableWriteback && !hasReadPerm(h.flags) {
 		err = syscall.EBADF
 		return
 	}

